README.md

```diff
@@ -45,36 +45,36 @@ Note that politeiawww does not provide HTML output.  It strictly handles the
 JSON REST RPC commands only.  The GUI for politeiawww can be found at:
 https://github.com/decred/politeiagui
 
-## Development 
+## Development
 
 #### 1. Install [Go](https://golang.org/doc/install) and [dep](https://github.com/golang/dep), if you haven't already.
-#### 2. Clone this repository. 
-#### 3. Setup configuration files: 
-* `politeiad` and `politeiawww` both have configuration files that you should 
-set up to make execution easier. 
+#### 2. Clone this repository.
+#### 3. Setup configuration files:
+* `politeiad` and `politeiawww` both have configuration files that you should
+set up to make execution easier.
 * You should create the configuration files under the following path:  
   - **Mac OS** `/Users/<username>/Library/Application Support/Politeiad/politeiad.conf`
   - **Windows** `C:\Users\<username>\AppData\Local\Politeiad/politeiad.conf`
   - **Ubuntu** `~/.politeiad/politeiad.conf`
-  
-* Copy and change the  [`sample-politeiawww.conf`](https://github.com/decred/politeia/blob/master/politeiawww/sample-politeiawww.conf) 
+
+* Copy and change the  [`sample-politeiawww.conf`](https://github.com/decred/politeia/blob/master/politeiawww/sample-politeiawww.conf)
 and [`sample-politeiad.conf`](https://github.com/decred/politeia/blob/master/politeiad/sample-politeiad.conf) files.
 * You may use the following default configurations:
 
-**politeiad.conf**: 
+**politeiad.conf**:
 
-    rpcuser=user 
-    rpcpass=pass 
-    testnet=true 
+    rpcuser=user
+    rpcpass=pass
+    testnet=true
 
 
 **politeiawww.conf**:
 
-    rpchost=127.0.0.1 
-    rpcuser=user 
-    rpcpass=pass 
-    rpccert="/Users/<username>/Library/Application Support/Politeiad/https.cert" 
-    proxy=true 
+    rpchost=127.0.0.1
+    rpcuser=user
+    rpcpass=pass
+    rpccert="/Users/<username>/Library/Application Support/Politeiad/https.cert"
+    proxy=true
     testnet=true
 
 **Attention:** The rpccert path is referenced on MacOs path. See above for more
```

Comment:
Could you add a note here, like: "politeiawww uses an email server to send verification codes for things like new user registration, and those settings are also configured within `politeiawww.conf`. See below for more information." and link the section at the bottom?